src/main/java/com/ca/apim/gateway/cagatewayconfig/tasks/zip/builder/BuilderUtils.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2018 CA. All rights reserved.
+ * This software may be modified and distributed under the terms
+ * of the MIT license.  See the LICENSE file for details.
+ */
+
+package com.ca.apim.gateway.cagatewayconfig.tasks.zip.builder;
+
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+
+import java.util.Map;
+
+class BuilderUtils {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Add a private constructor to hide the implicit public one. ([squid:S1118](https://sonarcloud.io/organizations/ca-api-gateway/rules#rule_key=squid%3AS1118))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=com.ca.apim.gateway%3Agateway-developer-plugin&pullRequest=52&issues=AWSQdW1usk71k7lwrENy&open=AWSQdW1usk71k7lwrENy)

[ISSUE_KEY]: AWSQdW1usk71k7lwrENy